R/model.R

```diff
@@ -1416,13 +1416,13 @@ CmdStanModel$set("public", name = "generate_quantities", value = generate_quanti
 #' @template seealso-docs
 #' @inherit CmdStanDiagnose examples
 #'
-diagnose_method <- function(data = NULL,
-                            seed = NULL,
-                            init = NULL,
-                            output_dir = NULL,
-                            output_basename = NULL,
-                            epsilon = NULL,
-                            error = NULL) {
+diagnose <- function(data = NULL,
```

Comment:
unrelated but should be `diagnose` instead of `diagnose_method` to stick with current conventions. this fixes an issue in the docs where `?diagnose` went to a page listing a method `'diagnose_method'`.

Reply:
Agreed.